website/docs/usage/large-language-models.mdx

```diff
@@ -0,0 +1,495 @@
+
+### Cache {id="cache"}
+
+Interacting with LLMs, either through an external API or a local instance, is
+costly. Since developing an NLP pipeline generally means a lot of exploration
+and prototyping, `spacy-llm` implements a built-in
+[cache](/api/large-language-models#cache) to avoid reprocessing the same
+documents at each run that keeps batches of documents stored on disk.
+
+### Various functions {id="various-functions"}
+
+| Component                                                               | Description                                                                                                                                                                                                                                                                          |
+| ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| [`spacy.FewShotReader.v1`](/api/large-language-models#fewshotreader-v1) | This function is registered in spaCy's `misc` registry, and reads in examples from a `.yml`, `.yaml`, `.json` or `.jsonl` file. It uses [`srsly`](https://github.com/explosion/srsly) to read in these files and parses them depending on the file extension.                        |
+| [`spacy.FileReader.v1`](/api/large-language-models#filereader-v1)       | This function is registered in spaCy's `misc` registry, and reads a file provided to the `path` to return a `str` representation of its contents. This function is typically used to read [Jinja](https://jinja.palletsprojects.com/en/3.1.x/) files containing the prompt template. |
+| [Normalizer functions](/api/large-language-models#normalizer-functions) | These functions provide simple normalizations for string comparisons, e.g. between a list of specified labels and a label given in the raw text of the LLM response.                                                                                                                 |
+
+## Ongoing work {id="ongoing-work"}
+
+In the near future, we will
+
+- Add more example tasks
+- Support a broader range of models
+- Provide more example use-cases and tutorials
+- Make the built-in tasks easier to customize via Jinja templates to define the
+  instructions & examples
+
+PRs are always welcome!
+
+## Reporting issues {id="issues"}
+
+If you have questions regarding the usage of `spacy-llm`, or want to give us
+feedback after giving it a spin, please use the
+[discussion board](https://github.com/explosion/spaCy/discussions). Bug reports
+can be filed on the
+[spaCy issue tracker](https://github.com/explosion/spaCy/issues). Thank you!
```

Comment:
This looks to me more like "readme" sections that things we should have in the spacy.io docs

Reply:
Yes, we can ditch this. This will remain in the readme.